src/main/controlMod.F90

```diff
@@ -866,6 +870,7 @@ subroutine control_print ()
     write(iulog,*) '    use_nitrif_denitrif = ', use_nitrif_denitrif
     write(iulog,*) '    use_extralakelayers = ', use_extralakelayers
     write(iulog,*) '    use_vichydro = ', use_vichydro
+    write(iulog,*) '    use_excess_ice = ', use_excess_ice !MVD to check if the value is read from namelists
```

Comment:
Remove !MVD

Reply:
Fixed

Reply:
Awesome.

Reply:
Actually I still see it there, so it would be good to remove the comment..